scripts/find_type.py

```diff
@@ -0,0 +1,67 @@
+# Usage: find_type.py FILENAME START_LINE START_COL END_LINE END_COL MYPY_AND_ARGS
```

Comment:
I had to read the code carefully to find out whether MYPY_AND_ARGS is supposed to be a single argument (with spaces embedded) or just the remaining items from sys.argv -- it's the latter, but can you clarify that?